lib/beaker/options/parser.rb

```diff
@@ -216,7 +217,12 @@ def parse_args(args = ARGV)
         cmd_line_options[:command_line] = ([$0] + args).join(' ')
         @attribution = @attribution.merge(tag_sources(cmd_line_options, "flag"))
 
-        # Subcommands are the first to get merged into presets
+        # Global options from $HOME/.beaker/subcommands_options.yaml are first to get merged into presets
+        homedir_options = Beaker::Options::OptionsFileParser.parse_homedir_options
```

Comment:
Just thinking out loud, I'm wondering if we should refactor SubcommandOptionsParser to accept a file path, and use that here? Since presumably we don't want to parse these options on init, which the SubcommandOptionsParser takes care of